app/models/session.rb

```diff
@@ -0,0 +1,50 @@
+class Session
+  include ActiveModel::Model
+  include Virtus.model
+  HOURS_TO_EXPIRE = 10
```

Comment:
This could be configurable (env var), side question how are we handling env vars locally? I've generally had a secure_config.yml.dist file which is part of git and contains no secure info. Locally you change that file to a non git committed secure_config.yml and load it at startup. That file uses the env var if it's set or a default:

```yaml
development: &dev
  session:
    ttl: <%= ENV['SESSION_TTL'] || 10 %>
  redis:
    url: <%= ENV['REDIS_URL'] || 'redis://localhost:6379/0' %>
    namespace: <%= ENV['REDIS_MY_NAMESPACE'] || 'my-namespace' %>
    # ...
```